server/streamable_http.go

```
@@ -0,0 +1,810 @@
+	server             *MCPServer
+	baseURL            string
+	basePath           string
+	endpoint           string
+	sessions           sync.Map // Maps sessionID to ClientSession
+	srv                *http.Server
+	contextFunc        SSEContextFunc
+	sessionIDGenerator func() string
+	enableJSONResponse bool
+	eventStore         EventStore
+	standaloneStreamID string
+	streamMapping      sync.Map // Maps streamID to response writer
```

golangci-lint (1.64.8)

field `streamMapping` is unused

(unused)